### ISSUE TITLE
Implement get descriptor / device qualifier

### ISSUE THREAD

**Issue by nitrotm** [hathach/tinyusb#410]
Most of the structure is there, it's only a matter of add a new callback and starting a transfer.

in `process_get_descriptor`:

```
    case TUSB_DESC_DEVICE_QUALIFIER:
    {
      TU_LOG2(" Device Qualifier\r\n");

      uint16_t len = sizeof(tusb_desc_device_qualifier_t);

      return tud_control_xfer(rhport, p_request, (void*) tud_descriptor_device_qualifier_cb(), len);
    }
```

**Comment by hathach** [comment]
yeah, I am just being lazy since not many people need it. Will add it when having time.

**Comment by hathach** [comment]
fixed by #421 but It is only step 1 of the process, after getting device qualifier successfully, host will ask for other speed configuration. Which is not supported currently, Other speed configuration will only be added when there is a testable example/setup
